spec/__snapshots__/snippingTool.spec.tsx.snap

```diff
@@ -62,6 +62,13 @@ exports[`Snipping Tool should render correctly 1`] = `
       className="imageContainer"
     >
       <AnnotateArea
+        annotateAreaDimensions={
+          Object {
+            "height": 600,
```

Comment:
Why not 800 ?

Reply:
This is just the markup for a snapshot test. It is created automatically by Jest to be a copy of the HTML structure in a given time. The reason why it is 600 though is that when seting up state for the tool, the default values are:

  const [annotateAreaDimensions, setAnnotateAreaDimensions] = useState({
    height: 600,
    width: 800,
  });